media/js/src/treegrowth/graph.js

```diff
@@ -38,10 +39,12 @@
             'SoilM_5cm', 'AvgPAR_Den'
         ];
 
-        // Remove Site 1 STD DEV
-        data.splice(6, 1);
-        // Remove Site 2 STD DEV
-        data.splice(12, 1);
+        if (data.length === 19) {
```

Comment:
There are a few magic numbers here and at Ln230. Can these be moved into constants?